webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -106,6 +116,51 @@ class ResponsePage extends Component {
         };
     };
 
+    renderReviewResponse(review_response, section) {
+        const questions = section.review_questions.map(q => {
+            const a = review_response.scores.find(a => a.review_question_id === q.id);
+            if (q.type === "information") {
+                return <h4>{q.headline}</h4>
+            };
+
+            if (a) {
+                return <div>
+            <div key={q.question_id} className="question-answer-block">
+                <p><span className="question-headline">{q.headline}</span>
+                    {q.description && a && <span className="question-description"><br/>{q.description}</span>}
+                </p>
+                <h6><AnswerValue answer={a} question={q} /></h6>
+            </div>
+            </div>
+            }
+            
+            
+        });
+        return questions
+    };
+
+    // Render Reviews
+    renderCompleteReviews(){
+        if (this.state.reviewResponses) {
+                const reviews = this.state.reviewResponses.map((val, index) => {
+                    let num = index + 1;
+                    return <div className="section">
+                        {val.review_response.is_submitted === true &&
```

Comment:
If you're not showing unsubmitted reviews on the front-end (which makes sense), then you also shouldn't return them from the backend so we don't send unnecessary data over the network. Will be easy to filter those out if you make your backend API just return all the (submitted) reviews in one go.